Models/Project/MyPlaces.cs

```diff
@@ -0,0 +1,27 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace rexfinder_api.Models;
+
+public class MyPlaces
+{
+
+    //these items are set by our app
+    public int MyPlaceId { get; set; }
+    public bool Visited { get; set; }
```

Comment:
Do we need/want to set Visited to a default value of False here? Or does that happen int the dBContext?